AutoMapper.Data.Tests/DataReaderMapping.cs

```diff
@@ -395,9 +401,8 @@ public IDataReader BuildDataReaderWithNullableField()
         public When_mapping_a_data_reader_to_a_dto_with_nullable_enum()
         {
             Mapper.Initialize(cfg => {
-                MapperRegistry.Mappers.Insert(0, new DataReaderMapper());
-
-                cfg.CreateMap<IDataReader, DtoWithSingleNullableField>();
+                cfg.Mappers.Insert(0, new DataReaderMapper());
+                cfg.AddMemberConfiguration().AddMember<DataRecordMemberConfiguration>();

```

Comment:
Good idea. I'm working on another PR that includes an `ISourceToDestinationNameMapper` implementation in addition to the `IChildMemberConfiguration` implementation. I'll add extension methods and update the ReadMe in that PR.